### PR TITLE
modified index.html to serve as home page

### PR DESCRIPTION
**Pull request by bhargaviks**
Hi,
I have created a barebones home page, that consists of a navbar, dropdown, buttons, and a footer. 
This is how the [page](https://screenshot.googleplex.com/s3dk6BOXQrE.png) looks when you enter
This is how it looks when you click the[ dropdown](https://screenshot.googleplex.com/AcYxUTbjQm4.png)

Thank you.

### CODE REVIEW

**Comment by bhargaviks** [comment]
@googlebot I signed it!

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WeTime</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css">       <!-- Bulma.css is the css I used. so importing its CDN -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>          <!-- Ajax, Jquery -->
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>             <!-- Font awesome- used for icons-->
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
```
Comment (mbouadeus) [inline src/main/webapp/index.html:15]: I'm wondering if all the empty lines are necessary.

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  font-family: 'Open Sans';
+  font-size: 4em;
+}
+
+img {
+  padding: 5px;
+  border: 1px solid #ccc;
+}
+
+.tabs{
```
Comment (mbouadeus) [inline src/main/webapp/style.css]: Add space between .tabs and {

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    <section class = "section">
+      <div class="container">
+        <div class="tabs is-centered">
+          <ul><li><a>We hope you have a great time!</a></li></ul>
+        </div>
+      </div>
+    </section>
+    
+  </body>
+</html>
```
Comment (mbouadeus) [inline src/main/webapp/index.html]: no newline at EOF.

File: src/main/webapp/style.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  font-size: 0.5em;
+}
+
+.dropdown-item {
+    font-size: 3.5rem;
```
Comment (mbouadeus) [inline src/main/webapp/style.css]: tab with two spaces.

**Review by trinitydonohugh — APPROVED** [review]

**Review by mbouadeus — CHANGES_REQUESTED** [review]

File: src/main/webapp/index.html (hunk 1 above)
Comment (bhargaviks) [reply src/main/webapp/index.html:15]: Yes, I generally add them in my code to increase readability

Comment (mbouadeus) [reply src/main/webapp/index.html:15]: Ok. That's fine.

**Review by mbouadeus — APPROVED** [review]

File: src/main/webapp/index.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WeTime</title>
```
Comment (isaacwdavis) [inline src/main/webapp/index.html:8]: https://wetime.io/

Doesn't mean you can't use this name as well.

File: src/main/webapp/index.html [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css">       <!-- Bulma.css is the css I used. so importing its CDN -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>          <!-- Ajax, Jquery -->
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>             <!-- Font awesome- used for icons-->
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+
+    <!-- This is the first section of the page that contains the navbar containing the WeTime title and other home, log in buttons, etc..-->
```
Comment (isaacwdavis) [inline src/main/webapp/index.html]: nit: space between end of comment and -->

File: src/main/webapp/index.html [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </div>
+        </nav>
+      </div>
+    </section>
+
+    <!-- This section of the page contains the drop box that span the entire width of the page as well as the buttons to schedule and browse-->
```
Comment (isaacwdavis) [inline src/main/webapp/index.html]: nit: space between end of comment and -->, also punctuation

File: target/maven-archiver/pom.properties [hunk 8]
```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Tue Jul 21 22:13:32 UTC 2020
+#Wed Jul 22 00:42:38 UTC 2020
```
Comment (isaacwdavis) [inline target/maven-archiver/pom.properties:2]: You might want to add this to the .gitignore file. Thoughts?

File: target/portfolio-1/style.css [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    font-size: 3.5rem;
+    line-height: 2;
+}
+
+.dropdown-content {
+    background-color: #fff;
+    border-radius: 4px;
+    padding-left: 18rem;
+    padding-right: 18rem;
+}
```
Comment (isaacwdavis) [inline target/portfolio-1/style.css:25]: nit: no newline

**Review by isaacwdavis — APPROVED** [review]

**Review by pengxianzhe — COMMENTED** [review]
I think you want to ignore changes in target/ directory. Is it correct?

**Review by pengxianzhe — APPROVED** [review]

**Review by trinitydonohugh — APPROVED** [review]

**Pull request merged**